src/NovakSolutions/Infusionsoft/Model/Model.php

```diff
@@ -91,6 +91,8 @@ public function fromArray($data){
                     foreach($value as $object){
```

Comment:
FYI, I've thought about this a lot, and I think if I were to redo this section, I would normalize the field details and then only have one chunk of code that takes a normalized model / field definition to instantiate it.  I think I would do it that way.  But, we're already here :). Not recommending a change, just FYI if you ever have to do something like this.